src/operations/command.ts

```diff
@@ -111,7 +111,7 @@ export abstract class CommandOperation<T> extends AbstractCallbackOperation<T> {
     server: Server,
     session: ClientSession | undefined,
     cmd: Document
-  ): Promise<Document> {
+  ): Promise<any> {
```

Comment:
```suggestion
  ): Promise<T> {
```
Is there a reason we can't use the generic type?  Operations are parameterized by their return types.